Makefile

```diff
@@ -115,7 +115,13 @@ lint_yarn_workspaces: ## Lints Yarn workspace packages
 	scripts/validate-workspaces.js
 
 lint_asset_bundle_size: ## Lints JavaScript and CSS compiled bundle size
-	find app/assets/builds/application.css -size -270000c | grep .
+# This enforces an asset size budget to ensure that download sizes are reasonable and to protect
+# against accidentally importing large pieces of third-party libraries. If you're here debugging a
+# failing build, check to ensure that you've not added more JavaScript or CSS than necessary, and
+# you have no options to split that from the main bundles. If you need to increase this budget and
+# accept the fact that this will force end-users to endure longer load times, you should set the new
+# budget to within a few thousand bytes of the current production-compiled size.
```

Comment:
I'd expect we indent this comment? Like in the `optimize_svg` task?

Reply:
> I'd expect we indent this comment? Like in the `optimize_svg` task?

Hmm for some reason I didn't think Make liked the syntax, but I see what you're saying about `optimize_svg`. Let me try again.

Reply:
maybe a tabs vs spaces issue? Makefiles are sensitive to the difference

Reply:
Okay, yeah, it's quite likely it was a spacing issue. That or I didn't like how it would output the comment to the terminal (not a huge deal, but 🤷 ). It's not pretty with all the `@`, but 4bb104579 updates so that it's indented and doesn't output the comment.